code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1463,6 +1463,19 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		"You hear a slap.")
 		stop_wagging_tail(target)
 		return FALSE
+	var/aim_for_groin  = user.zone_selected == "groin"
+	var/target_aiming_for_groin = target.zone_selected == "groin"
+	if(aim_for_groin && (target_on_help_and_unarmed || target_restrained || target_aiming_for_groin))
+		playsound(target.loc, 'sound/weapons/slap.ogg', 50, 1, -1)
+		user.visible_message("<span class='danger'>[user] slaps [target]'s ass!</span>",
+			"<span class='notice'>You slap [target]'s ass! </span>",\
+		"You hear a slap.")
+		stop_wagging_tail(target)
```

Comment:
make this only stop wagging the tail if the target isn't a nymphomaniac

Reply:
Added, also added the nympho check for wagging to face slapping.